app/javascript/src/accessible-autocomplete.js

```diff
@@ -0,0 +1,42 @@
+import accessibleAutocomplete from 'accessible-autocomplete'
+
+const screenReaderMessageDelay = 1000 // wait before updating the screen reader message, to avoid interrupting queue
+const searchSelectItem = document.querySelector('.country-select')
+
+function enhanceSelectElement (searchSelectItem) {
+  accessibleAutocomplete.enhanceSelectElement({
+    defaultValue: '',
+    selectElement: searchSelectItem,
+    inputClasses: 'govuk-input',
+    name: 'country'
+  })
+
+  addSearchInputListeners()
+
+  if (document.querySelector('#non-uk-home-address-country-code-error')) {
+    addErrorClasses()
+  }
+}
+
+function addSearchInputListeners () {
+  const searchInputBox = document.querySelector('.autocomplete__input')
+
+  document.querySelector('.clear-search').style.display = 'block'
```

Comment:
Separation of concerns (SoC): This would add inline css to the html.
Would it be ideal to add/toggle a class name instead?